tests/phpunit/FileSystem/FileSystemTestCase.php

```diff
@@ -35,62 +35,42 @@
 
 namespace Infection\Tests\FileSystem;
 
-use function getenv;
-use function Infection\Tests\make_tmp_dir;
+use Infection\Tests\TestingUtility\FS;
 use PHPUnit\Framework\TestCase;
 use function Safe\getcwd;
 use function Safe\realpath;
-use function sprintf;
+use function str_replace;
 use Symfony\Component\Filesystem\Filesystem;
-use Symfony\Component\Filesystem\Path;
-use function sys_get_temp_dir;
 
 /**
  * @private
  */
 abstract class FileSystemTestCase extends TestCase
 {
-    private const TMP_DIR_NAME = 'infection-test';
-
     protected string $cwd = '';
 
     protected string $tmp = '';
 
-    public static function tearDownAfterClass(): void
-    {
-        // Cleans up whatever was there before. Indeed upon failure PHPUnit fails to trigger the
-        // `tearDown()` method and as a result some temporary files may still remain.
-        self::removeTmpDir();
```

Comment:
could you please explain why we removed it? What will happen if PHPUnit fails (as described in the comment)?

Reply:
the `::tearDown()` method does get called, even if the test fail. I don't know if it was an earlier version of PHP and/or due to the introduction of `Throwable` or smth and it changed at some point. But that comment is not accurate anymore.